webui/cypress/integration/validate-tests.spec.js

```diff
@@ -0,0 +1,82 @@
+//Test check validate in:
+//1. Create replicaset dialog
+//2. Probe server dialog
+//3. Edit replicaset dialog
+//4. Add user dialog
+
+describe('Replicaset configuration & Bootstrap Vshard', () => {
+
+  it('1. Create replicaset dialog', () => {
+    cy.visit(Cypress.config('baseUrl'));
+
+    cy.get('.meta-test__configureBtn').first().click();//component: UnconfiguredServerList
+    cy.get('.meta-test__ConfigureServerModal input[name="alias"]')
+      .type(' ');
+    cy.get('.meta-test__ConfigureServerModal').contains('Alias must contain only alphanumerics');
+
+    cy.get('.meta-test__ConfigureServerModal input[name="alias"]')
+      .type('{selectall}{backspace}');
+    cy.get('.meta-test__ConfigureServerModal').contains('Alias must contain only alphanumerics').should('not.exist');
+    
+    cy.get('.meta-test__ConfigureServerModal input[name="alias"]')
+      .type('for-validate-tests');
+    cy.get('.meta-test__ConfigureServerModal button[type="button"]').contains('Select all').click();
+
+    cy.get('.meta-test__ConfigureServerModal input[name="weight"]')
+      .type('q');
+    cy.get('.meta-test__ConfigureServerModal').contains('Field accepts number');
+
+    cy.get('.meta-test__ConfigureServerModal input[name="weight"]')
+      .type('{selectall}{backspace}');
+    cy.get('.meta-test__ConfigureServerModal').contains('Field accepts number').should('not.exist');
+
+    cy.get('.meta-test__CreateReplicaSetBtn').click();
+  })
+
+  it('2. Probe server dialog',() => {
+    cy.get('.meta-test__ProbeServerBtn').click();
+    cy.get('.ProbeServerModal input[name="uri"]')
+      .type(' ');
+    cy.get('.meta-test__ProbeServerSubmitBtn').click();//component:ProbeServerModal
+    cy.get('.ProbeServerModal').contains('Probe " " failed: parse error');
+    cy.get('.ProbeServerModal input[name="uri"]')
+      .type('local');
```

Comment:
Let"s extend test cases
* press escape, check that dialog is closed
* Ping localhost:9 (or some other unreacable host), Check that error message says "no response" (dialog should still be open)
* Enter uri already joined, press enter, check that the dialog is closed and that success is notified